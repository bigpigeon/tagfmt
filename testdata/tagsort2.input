//tagfmt -s -so "json|yaml|desc"
package main
type Example struct {
	Data string `desc:"some inuse data" yaml:"data" json:"data" `
}