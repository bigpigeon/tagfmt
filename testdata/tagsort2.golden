//tagfmt -s -so "json|yaml|desc"
package main

type Example struct {
	Data string `json:"data" yaml:"data" desc:"some inuse data"`
}
